src/ParseServer.js

```diff
@@ -83,10 +83,22 @@ class ParseServer {
     // Note: Tests will start to fail if any validation happens after this is called.
     databaseController
       .performInitialization()
-      .then(() => hooksController.load())
+      .then(async () => await hooksController.load())
```

Comment:
I don't see the point in the changes here, doesn't returning a promise should be the same as awaiting one?

Reply:
Adding explicit async await calls allow to preserve.the stack trace and improve debugging.

If your nested promise is simply returned to a grand parent, all middle stack trace will be lost. Which can get confusing for users.

Reply:
Right, thank you for that information @sadortun! Glad to have learnt something 😊 .

In that case, could it be beneficial to refactor to:

(async () => {
     await databaseController.performInitialization()
     await hooksController.load()
})();

Or is this pointless?

Reply:
> could it be beneficial to refactor to:

Yes! But we should really remove ( later ) any async stuff from the constructor. For now, the goal of this issue is just tu put what's required for my Schema/Migration PR.

Actually I'll change it to : 
```js
Promise.resolve().then(async ()=> databaseController.performInitialization())
```

Reply:
That should good to me. Perhaps that can be marked as part of our overall modernisation of the codebase, which we can focus on after V5.